python/kserve/kserve/model_server.py

```diff
@@ -165,8 +167,10 @@ def __init__(
         enable_grpc: bool = args.enable_grpc,
         enable_docs_url: bool = args.enable_docs_url,
         enable_latency_logging: bool = args.enable_latency_logging,
-        configure_logging: bool = args.configure_logging,
-        log_config: Optional[Union[Dict, str]] = args.log_config_file,
+        configure_logging: bool = None,
```

Comment:
We did not change the default value on the argument `--configure_logging` though, so user may still expect this to work even we mark it `deprecated`